plugins/catalog-react/src/components/DefaultFilters/DefaultFilters.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  EntityKindPicker,
+  EntityTypePicker,
+  UserListPicker,
+  EntityOwnerPicker,
+  EntityLifecyclePicker,
+  EntityTagPicker,
+  EntityProcessingStatusPicker,
+  EntityNamespacePicker,
+  UserListFilterKind,
+  EntityOwnerPickerProps,
+} from '@backstage/plugin-catalog-react';
```

Comment:
these should be local paths right? it's all from inside the same package it seems

Reply:
Yes, when I created the `DefaultFilters` I did that in `@backstage/plugin-catalog` and thought it made more sense to be in `@backstage/plugin-catalog-react`. I'll fix these up 👍

Reply:
Imports have been fixed 👍